modules/ROOT/pages/dataweave-system-properties.adoc

```diff
@@ -60,97 +36,91 @@ output application/dw
+* Type: `Boolean`
+* Default: `false`
 //com.mulesoft.dw.directbuffer.disable//
 |`com.mulesoft.dw.directbuffer.disable`|
 Controls whether DataWeave uses off-heap memory (the default) or heap memory. DataWeave uses off-heap memory for internal buffering, which can cause issues on systems that have only a small amount of memory. See xref:dataweave-memory-management.adoc[DataWeave Memory Management].
-
 * Type: `Boolean`
 * Default: `false`
-
-
 //com.mulesoft.dw.dump_files//
 |`com.mulesoft.dw.dump_files`|
-_Experimental:_ Dumps the input context and the failing script into a folder when this property is set to `true`. This behavior enables you to track the failing script and the data that makes the script fail, which is particularly useful for verifying that the received input data is valid. Incorrect scripts often fail when an upstream component generates invalid data.
-
+_Experimental:_ Dumps the input context and the failing script into a folder when this property is set to true. This behavior enables you to track the failing script and the data that makes the script fail, which is particularly useful for verifying that the received input data is valid. Incorrect scripts often fail when an upstream component generates invalid data.
 * Type: `Boolean`
 * Default: `false`
-
 `com.mulesoft.dw.dump_files` is an _experimental feature_ that is subject to change or removal from future versions of DataWeave. See xref:dataweave-troubleshoot.adoc[Troubleshooting a Failing DataWeave Script].
-
-
 //com.mulesoft.dw.dump_folder//
 |`com.mulesoft.dw.dump_folder`|
 _Experimental:_ Specifies the path in which to dump files when `com.mulesoft.dw.dump_files` is set to `true`.
-
 * Type: `String`
-* Default: `java.io.tmpdir` (system property value)
-
+* Default: `/var/folders/8f/5jqs1kh97k16dzsmxxph2m5c0000gp/T`
 `com.mulesoft.dw.dump_folder` is an _experimental feature_ that is subject to change or removal from future versions of DataWeave. See xref:dataweave-troubleshoot.adoc[Troubleshooting a Failing DataWeave Script].
-
-
 //com.mulesoft.dw.dumper_fill_stacktrace//
 |`com.mulesoft.dw.dumper_fill_stacktrace`|
 _Experimental:_ Adds the StackTrace for dumper exceptions when `com.mulesoft.dw.dump_files` is set to `true`.
-
 * Type: `Boolean`
 * Default: `false`
-
-`com.mulesoft.dw.dumper_fill_stacktrace` is an _experimental feature_ that is subject to change or removal from future versions of DataWeave.
-
-
-//com.mulesoft.dw.error_value_length//
-|`com.mulesoft.dw.error_value_length`|
-Sets the maximum length of exception messages to display to the user. The message is truncated to the maximum length. This setting is useful for avoiding long exception messages.
-
-* Type: `Number`
-* Default: `80`
-
-
+`com.mulesoft.dw.dumper_fill_stacktrace` is an _experimental feature_ that is subject to change or removal from future versions of DataWeave. See xref:dataweave-troubleshoot.adoc[Troubleshooting a Failing DataWeave Script].
```

Comment:
```suggestion
`com.mulesoft.dw.dumper_fill_stacktrace` is an _experimental feature_ that is subject to change or removal from future versions of DataWeave. See xref:dataweave-troubleshoot.adoc[].
```

Reply:
The titles get auto-filled. Empty brackets are preferred for maintenance purposes in case the title changes. In other cases, where we want to type something other than the title itself for some reason, then we can put text between the brackets.

Reply:
added to master